components/bl-stripe-payment-component/src/payment-components/payment-amount.js

```diff
@@ -1,6 +1,6 @@
 import { useMemo } from 'react';
 
-const ONLY_NUMERIC_REGEX = /[-+eE]/;
+const ONLY_NUMERIC_REGEX = /^[\d.,]+$/;
```

Comment:
try this pattern `^\d+[.,]?\d+$`

you can test it here https://regex101.com/#javascript